v-next/hardhat/test/internal/cli/main.ts

```diff
@@ -415,6 +415,17 @@ describe("main", function () {
         });
       });
 
+      it("should throw because the task parameter is not defined", function () {
```

Comment:
I think this title is probably a copy-paste accident.
```suggestion
      it("should return the task id if not found", function () {
```